settings/OpeningPeriodForm/OpeningPeriodFormWrapper.js

```diff
@@ -130,10 +130,21 @@ class OpeningPeriodFormWrapper extends React.Component {
     });
   }
 
+  catchOverlappedEvents = (err) => {
+    if (err.status === 422) {
+      this.setState({
+        errorModalText: <FormattedMessage id="ui-calendar.dublication" />,
+      });
+      this.render();
```

Comment:
Could you please explain the need for calling `render` this way? Usually, render is not called that explicitly. Rather implicitly as result of props or state changes.